app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,198 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+    If you train to teach physics, you could also be eligible receive a scholarship or bursary of up to £29,000.
+  </p>
+  </section>
+
+   <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Spark curiosity in physics",
+      colour: "yellow",
+      image_1_path: "static/content/hero-images/0012.jpg",
+      image_1_alt: "A teacher sitting with a student helping them with a maths problem",
+      image_2_path: "static/content/hero-images/0007.jpg",
+      image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
+    ) do %>
+       <p>
+    As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.
+  </p>
+
+  <p>
+    You can inspire your students to consider rewarding careers from tackling climate change to artificial intelligence.
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the national curriculum, with opportunities to develop your students’ ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover:
+      </p>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
```

Comment:
there is a BIG gap here, particularly apparent on mobile:

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/4b3bf7af-9a6a-41b6-8cda-48964e4f93a9)

Reply:
I agree, but this is an issue throughout the site - huge amounts of whitespace between components and sections which pushes the content further down the page and doesn't make best use of the space available, particularly on mobile.

Reply:
Yes @garethwilliamsdesign but I think there is a particular issue here on this page as this is a much bigger gap than anywhere else